test/edu-benefits/1990n/01-all-fields.e2e.spec.js

```diff
@@ -0,0 +1,95 @@
+
+    // Applicant information page
+    client.waitForElementVisible('input[name="root_veteranFullName_first"]', Timeouts.slow);
+    Edu1990nHelpers.completeVeteranInformation(client, testData.applicantInformation.data);
+    client
+      .axeCheck('.main')
+      .click('.form-progress-buttons .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/applicant/information');
+
+    // Additional benefits page
+    client.waitForElementVisible('label[for="root_civilianBenefitsAssistance"]', Timeouts.slow);
+    EduHelpers.completeAdditionalBenefits(client, testData.additionalBenefits.data, false);
+    client
+      .axeCheck('.main')
+      .click('.form-progress-buttons .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/applicant/additional-benefits');
+
+    // Applicant service page
+    client.waitForElementVisible('label[for="root_view:applicantServed"]', Timeouts.slow);
+    EduHelpers.completeServicePeriods(client, testData.applicantService.data, false, 'view:applicantServed');
+    client
+      .axeCheck('.main')
+      .click('.form-progress-buttons .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/applicant/service');
+
+    // Benefits selection page
+    client.waitForElementVisible('label[for="root_payHighestRateBenefit"]', Timeouts.slow);
+    EduHelpers.completeBenefitsSelection(client, testData.benefitSelection.data);
+    client
+      .axeCheck('.main')
+      .click('.form-progress-buttons .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/benefits/eligibility');
+
+    // School selection page
+    client.waitForElementVisible('input[name="root_educationProgram_name"]', Timeouts.slow);
+    EduHelpers.completeSchoolSelection(client, testData.schoolSelection.data, true);
+    client
+      .axeCheck('.main')
+      .click('.form-progress-buttons .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/school-selection');
+
+    // Contact information page
+    client.waitForElementVisible('label[for="root_preferredContactMethod"]', Timeouts.slow);
+    EduHelpers.completeContactInformation(client, testData.contactInformation.data, false);
+    client
+      .axeCheck('.main')
+      .click('.form-progress-buttons .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/personal-information/contact-information');
+
+    // Direct deposit page
+    client.waitForElementVisible('label[for="root_bankAccount_accountType"]', Timeouts.slow);
+    EduHelpers.completeDirectDeposit(client, testData.directDeposit.data, false);
+    client
+      .axeCheck('.main')
+      .click('.form-progress-buttons .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/personal-information/direct-deposit');
+
+    // Review and submit page
+    client
+      .waitForElementVisible('label[name="privacyAgreement-label"]', Timeouts.slow)
+      .pause(1000)
+      .click('input[type="checkbox"]')
```

Comment:
Weird that it's not progressing to the confirmation page. One thing to verify is that the privacy checkbox is indeed checked since it's the only thing being validated here that the other pages haven't already validated.